components/list/list-item-generic-layout.js

```diff
@@ -171,6 +174,10 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				grid-column: content-start / end;
 				grid-row: nested-start / nested-end;
 			}
+			::slotted([slot="add"]) {
+				grid-column: color-start / end;
+				grid-row: add-start / add-end;
```

Comment:
It's not necessary to change this, but when the names are `add-start` and `add-end`, the "start" and "end" will be assumed when writing `grid-row: add;`. Just a little short-hand tip! 

I started using this short-hand [here](https://github.com/BrightspaceUI/lms-core/blob/main/src/components/attachments/attachment.js#L54) if you're curious to see it in action.